purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -80,21 +80,29 @@ open class PurchasesConfiguration(builder: Builder) {
         }
 
         /**
-         * **Feature in beta**. Defines how strict [EntitlementInfo] verification ought to be.
+         * **Feature in beta**. Enables signature verification of requests to the RevenueCat backend
+         * in informational mode and enables diagnostics reports to RevenueCat to help us analyze this feature.
+         * Informational mode means that the behavior will remain the same but we will provide information about
+         * detected attempts of hacking.
          *
-         * When changing from [EntitlementVerificationMode.DISABLED] to [EntitlementVerificationMode.INFORMATIONAL],
-         * the SDK will clear the CustomerInfo cache. This means users will need to connect to the internet to get
-         * back their entitlements.
+         * When changing from disabled to enabled, the SDK will clear the CustomerInfo cache.
+         * This means users will need to connect to the internet to get back their entitlements.
```

Comment:
Should this be "This means users will need to connect to the internet to get their entitlements back."?